src/Cake.Issues.Tests/IIssueComparerTests.cs

```diff
@@ -783,6 +783,53 @@ public void Should_Return_True_If_ProviderType_Is_Same()
                 CompareIssues(issue1, issue2, true);
             }
 
+            [Fact]
+            public void Should_Return_True_If_AdditionalInformation_Is_Same()
+            {
+                // Given
+                var issue1 =
+                    IssueBuilder
+                        .NewIssue("message", "providerType", "providerName")
+                        .WithAdditionalInformation("Foo", "FooValue")
+                        .WithAdditionalInformation("Bar", "BarValue")
+                        .WithAdditionalInformation("FooBar", "FooBarValue")
+                        .Create();
+                var issue2 =
+                    IssueBuilder
+                        .NewIssue("message", "providerType", "providerName")
+                        .WithAdditionalInformation("Foo", "FooValue")
+                        .WithAdditionalInformation("Bar", "BarValue")
+                        .WithAdditionalInformation("FooBar", "FooBarValue")
+                        .Create();
+
+                // When / Then
+                CompareIssues(issue1, issue2, true);
+            }
+
+            [Fact]
+            public void Should_Return_False_If_AdditionalInformation_Is_Not_Same()
+            {
+                // Given
+                var issue1 =
+                    IssueBuilder
+                        .NewIssue("message", "providerType", "providerName")
+                        .WithAdditionalInformation("Foo", "FooValue")
+                        .WithAdditionalInformation("Bar", "BarValue")
+                        .WithAdditionalInformation("FooBar", "FooBarValue")
+                        .Create();
+                var issue2 =
+                    IssueBuilder
+                        .NewIssue("message", "providerType", "providerName")
+                        .WithAdditionalInformation("Foo", "FooValue")
+                        .WithAdditionalInformation("Bar", "BarValueWrong")
+                        .WithAdditionalInformation("FooBar", "FooBarValue")
```

Comment:
This should be different than in issue1, if the the test should fail. I would make it very clear with "ValueHarry" and "ValueSally" and not for example "FooValue1" and "FooValue2". Use of variables makes things easier. If you declare a variable like

var wrongValue = "MyBad";
var goodValue = "OK";

You can then use those variables without caring about typos. It is also easier to read and review. As soon as you are using two strings in a test, I would declare variables.

Reply:
Should be better now.